src/chocolatey/infrastructure/cryptography/CryptoHashProvider.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright � 2011 - Present RealDimensions Software, LLC
+// Copyright � 2011 - Present RealDimensions Software, LLC

```

Comment:
Looks like you dropped this from UTF-8. Very much dislike when Visual Studio does this. 👎